packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -1013,6 +1020,12 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
         }
         this._scaleBoxesDragBehaviors.length = 0;
         this._rotateAnchorsDragBehaviors.length = 0;
+        this.onDragStartObservable.clear();
+        this.onBoxOrAnchorHoverObservable.clear();
+        this.onScaleBoxDragObservable.clear();
+        this.onScaleBoxDragEndObservable.clear();
+        this.onRotationSphereDragObservable.clear();
+        this.onRotationSphereDragEndObservable.clear();

```

Comment:
were those simply missed? So this is also a bug fix? ;-)

Reply:
Yes, a bug fix as well after sebavan reminded me to clear the new on on dipose x)